src-graphql/attribute-query.js

```diff
@@ -0,0 +1,5 @@
+import baseQuery from './base-query';
+
+export default function attributeQuery(fields = ['key', 'value']) {
```

Comment:
It can be used to query on checkouts as well (which I actually forgot to add and will add now). Maybe `customAttributeQuery` since that's what the field name is called on the [schema](https://github.com/Shopify/shopify/blob/master/db/graphql/storefront_schema.graphql#L28)?